go.mod

```
@@ -1,6 +1,8 @@
 module github.com/datumforge/datum
 
-go 1.22.2
+go 1.22.3
+
+toolchain go1.22.4
```

```suggestion
go 1.22.4
```

I think you should just be able to do that ^ and do a `tidy`